src/main.rs

```diff
@@ -672,7 +672,26 @@ fn exec_cargo_inner(
     if progress.count != 0 && !cx.print_command_list && cx.log_group == LogGroup::None {
         eprintln!();
     }
-    progress.count += 1;
+
+    let new_count = progress.count + 1;
+    let mut skip = false;
+    if let Some(partition) = &cx.partition {
+        if progress.count % partition.count != partition.index - 1 {
```

Comment:
this again subjective as well. but it's more intuitive not to see adjusted progress after partitioning.